src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Tooltip/RazorClassifiedTextElement.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using Newtonsoft.Json;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer.Tooltip
+{
+    /// <summary>
+    /// Equivalent to VS' ClassifiedTextElement. The class has been adapted here so we
+    /// can use it for LSP serialization since we don't have access to the VS version.
+    /// Refer to original class for additional details.
+    /// </summary>
+    internal sealed class RazorClassifiedTextElement
```

Comment:
Instead of `Razor` i'd just mark all the classification types as `VSxyz` to be extra explicit